main/export.js

```diff
@@ -113,7 +113,7 @@ class Export {
         ...this.exportOptions,
         defaultFileName: fileType ? `${path.parse(this.defaultFileName).name}.${fileType}` : this.defaultFileName,
         inputPath: this.inputPath,
-        onProgress: percentage => this.setProgress('Converting…', percentage)
+        onProgress: (percentage, estimate) => this.setProgress(estimate ? `Converting (${estimate} remaining)` : 'Converting…', percentage)
```

Comment:
Can you change this to https://wulkano.slack.com/archives/G20K13PS7/p1567836784038400 ?